fe/estimator.py

```diff
@@ -139,29 +148,32 @@ def _deltaG(model, sys_params) -> Tuple[Tuple[float, List], np.array]:
         bp = unbound_pot.bind(np.asarray(params))
         bound_potentials.append(bp)
 
+    func_args = []
+    for lamb in model.lambda_schedule:
```

Comment:
I pulled this out as I found it clumsy to modify two function calls any time I changed the function signature

Reply:
Nice!

Another way we might refactor this is to avoid the need for two code paths predicted on `model.client is None`, by adding a `NullClient` or similar in [parallel/client.py](https://github.com/proteneer/timemachine/blob/e8773e6456cdc70141ab9d17e2773d4fc1f2f111/parallel/client.py) ?

Reply:
+1

Reply:
Nicely resolved in https://github.com/proteneer/timemachine/commit/4def6196c6a5d75116c76f830bd8910bfb0e2c58